tools/releases/distros.sh

```diff
@@ -37,41 +38,44 @@ function create_tarball() {
   local dest_dir=build/$RELEASE_NAME-$distro-$arch
   local kuma_dir=$dest_dir/$RELEASE_NAME-$KUMA_VERSION
 
-  rm -rf $dest_dir
-  mkdir $dest_dir
-  mkdir $kuma_dir
-  mkdir $kuma_dir/bin
-  mkdir $kuma_dir/conf
+  rm -rf "$dest_dir"
+  mkdir "$dest_dir"
+  mkdir "$kuma_dir"
+  mkdir "$kuma_dir"/bin
+  mkdir "$kuma_dir"/conf
 
-  get_envoy $distro $envoy_distro
-  chmod 755 build/envoy-$distro
+  get_envoy "$distro" "$envoy_distro"
+  chmod 755 build/envoy-"$distro"
 
-  cp -p build/envoy-$distro $kuma_dir/bin/envoy
-  cp -p build/artifacts-$system-$arch/kuma-cp/kuma-cp $kuma_dir/bin
-  cp -p build/artifacts-$system-$arch/kuma-dp/kuma-dp $kuma_dir/bin
-  cp -p build/artifacts-$system-$arch/kumactl/kumactl $kuma_dir/bin
-  cp -p build/artifacts-$system-$arch/coredns/coredns $kuma_dir/bin
-  cp -p build/artifacts-$system-$arch/kuma-prometheus-sd/kuma-prometheus-sd $kuma_dir/bin
-  cp -p $KUMA_CONFIG_PATH $kuma_dir/conf/kuma-cp.conf.yml
+  cp -p build/envoy-"$distro" "$kuma_dir"/bin/envoy
+  cp -p build/artifacts-"$system"-"$arch"/kuma-cp/kuma-cp "$kuma_dir"/bin
+  cp -p build/artifacts-"$system"-"$arch"/kuma-dp/kuma-dp "$kuma_dir"/bin
+  cp -p build/artifacts-"$system"-"$arch"/kumactl/kumactl "$kuma_dir"/bin
+  cp -p build/artifacts-"$system"-"$arch"/coredns/coredns "$kuma_dir"/bin
+  cp -p build/artifacts-"$system"-"$arch"/kuma-prometheus-sd/kuma-prometheus-sd "$kuma_dir"/bin
```

Comment:
```suggestion
  cp -p "build/artifacts-${system}-${arch}/kuma-prometheus-sd/kuma-prometheus-sd" "${kuma_dir}/bin"
```

maybe this style of quoting is a bit clearer? :shrug:

Reply:
fixed this and other occurrences in the file